src/etc/inc/unbound.inc

```diff
@@ -376,6 +376,9 @@ include: {$g['unbound_chroot_path']}{$cfgsubdir}/host_entries.conf
 # dhcp lease entries
 include: {$g['unbound_chroot_path']}{$cfgsubdir}/dhcpleases_entries.conf
 
+# OpenVPN client entries
```

Comment:
Perhaps this should be wrapped in similar tests to the case in openvpn.inc, so it isn't looking for these files when they do not or cannot exist? Or at least test for `$config['unbound']['regovpnclients']`